java/src/main/java/com/google/cloud/dataproc/templates/jdbc/JDBCToGCS.java

```diff
@@ -65,17 +80,40 @@ public void runTemplate() {
               + "in resources/conf/template.properties file or at runtime. Refer to jdbc/README.md for more instructions.");
     }
 
+    if (StringUtils.isNotBlank(concatedPartitionProps)
```

Comment:
This logic makes parallel read as mandatory and job fails if partitioning is not specified.

Reply:
No. that is not correct.
It is a multiline condition If (concatedPartitionProps != blank && (jdbcSQLPartitionColumn != blank || ...)) then trow exception.